src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
/cc @AtsushiKan, @stephentoub 
Ok, let's use numbers then.

@karelz @danmosemsft - there are at least 31 occurrences throughout the code. Please open a tracking issue to change all of those to numbers only.